netman/adapters/switches/arista.py

```diff
@@ -76,6 +77,55 @@ def remove_vlan(self, number):
         if not vlan.delete(number):
             raise OperationNotCompleted("Unable to remove vlan {}".format(number))
 
+    def add_ip_to_vlan(self, vlan_number, ip_network):
+        vlan = self.get_vlan_interface_data(vlan_number)
+
+        ip_found = next((ip for ip in vlan.ips if ip.ip == ip_network.ip), False)
+        if ip_found:
+            raise IPAlreadySet(ip_network, ip_found)
+
+        has_ips = len(vlan.ips) > 0
+        add_ip_command = "ip address {} {}".format(str(ip_network), "secondary" if has_ips else "")
+
+        commands = [
+            "enable",
+            "configure terminal",
+            "interface VLAN{:04d}".format(vlan_number),
+            add_ip_command
+        ]
+        try:
+            self.conn.execute(commands)
```

Comment:
You can use .enable instead of .execute https://pyeapi.readthedocs.io/en/master/client_modules/client.html#pyeapi.client.Node.enable